code/src/types/cart.types.ts

```diff
@@ -27,3 +27,10 @@ export type CartType = {
   items: CartItem[];
   totalPrice: number;
 };
+
+export type CartItemProps = {
+  item: CartItem;
+  onRemove: (product: CartItem) => void;
+};
+
+export type CartDrawerItemProps = Pick<CartItemProps, "onRemove"> & CartItem;
```

Comment:
also here. Just move it to the file where this component is placed, because it's not a global type, not a separate entity, it is just props for the component.

Reply:
> also here. Just move it to the file where this component is placed, because it's not a global type, not a separate entity, it is just props for the component.

What do you think if I placed all types in one file related to cart. Is it not ok ?